### ISSUE TITLE
Visualize failed 

### ISSUE THREAD

**Issue by ShungJhon** [GuyTevet/MotionCLIP#20]
when I test the Text-to-Motion part, the visualize part failed, 
```
list index out of range
  File "/home/vatis/anaconda3/envs/zxz_mld_plt/lib/python3.9/multiprocessing/pool.py", line 870, in next
    raise value
  File "/home/vatis/DataDisk_1/22_vatis_master/zhouxiangzhong/MotionCLIP-main/src/visualize/visualize.py", line 111, in pool_job_with_desc
    for _ in pool.imap_unordered(plot_3d_motion_dico, iterator): 
  File "/home/vatis/DataDisk_1/22_vatis_master/zhouxiangzhong/MotionCLIP-main/src/visualize/visualize.py", line 139, in generate_by_video
    gener["frames"] = pool_job_with_desc(pool, iterator,
  File "/home/vatis/DataDisk_1/22_vatis_master/zhouxiangzhong/MotionCLIP-main/src/visualize/visualize.py", line 413, in viz_clip_edit
    frames = generate_by_video({}, {}, generation,
  File "/home/vatis/DataDisk_1/22_vatis_master/zhouxiangzhong/MotionCLIP-main/src/visualize/motion_editing.py", line 33, in main
    viz_clip_edit(model, datasets, edit_csv, epoch, parameters, folder=folder)
  File "/home/vatis/DataDisk_1/22_vatis_master/zhouxiangzhong/MotionCLIP-main/src/visualize/motion_editing.py", line 37, in <module>
    main()
  File "/home/vatis/anaconda3/envs/zxz_mld_plt/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/vatis/anaconda3/envs/zxz_mld_plt/lib/python3.9/runpy.py", line 197, in _run_module_as_main (Current frame)
    return _run_code(code, main_globals, None,
IndexError: list index out of range
```
it seems that` pool.imap_unordered(plot_3d_motion_dico, iterator) ` fails.

**Comment by Kebii** [comment]
I meet the same error when running the text-to-motion script.  Have you found a solution to this issue?

**Comment by Kebii** [comment]
The reason for this error is due to a mismatch in the version of matplotlib. I fixed it by reinstalling matplotlib==3.4.3 and matplotlib-inline==0.1.2. Thanks!
